www/src/pages/docs/50_StoreMapping.md

```diff
@@ -12,63 +12,72 @@ eleventyNavigation:
 
 ## Overview
 
-fritz2 uses stores to manage the application state by holding some data model. But quite often those data does 
-not fit naturally to the needed UI-fragment. This is often due to the normalized form, where redundancy is avoided
-as much as possible. Also, HTML can only render `Strings` in the end, but the `String` representation of some data 
-types might differ from case to case, and we do not want to store those all explicitly.
+fritz2 uses stores to manage the application state by holding a data model. But quite often the data does 
+not fit into the desired UI-fragment out of the box. This is often due to the normalized form which avoids redundancy 
+as much as possible. Also, HTML can only render `Strings` at the end of the day, but the correct `String` representation 
+of data types differs from case to case, and we do not want to store those explicitly.
 
-In order to support clean data management but also a good match between data and UI-shape, fritz2's store concept
-offers a powerful concept: Store *mapping*.
+In order to support clean data management and compromise well between data and UI-shape, fritz2's stores 
+offer a powerful concept: store mapping.
 
-Like the known `map`-function from collections, where some source type `T` gets transformed to some other type `R`
-inside an expression, we can also *map* a store in order to change its source type to some other, better fitting type.
-There is one big difference between the classical `map`-function and the store's mapping functions: A store needs
-not only a function from `T -> R` (getter) but also from `R -> T` (setter) as a store manages changes!
+Remember the `map`-function from collections where a source type `T` is transformed to another type `R`
+inside of an expression? We can also *map* a store in order to change its source type to a more suitable type.
+However, there is one big difference between the classical `map`-function and the store's mapping functions: A store needs
+both a getter-function from `T -> R` and a setter-function from `R -> T` to manage changes, while the collections functions
+only requires the getter.
 
 ### Lenses
 
-There is a universal concept in computer science for such a functionality called *lens*. You might have a look at
-the [excellent documentation on lenses](https://arrow-kt.io/learn/immutable-data/lens/) from the 
-[arrow-project](https://arrow-kt.io).
+If you are unfamiliar with the universal concept for the functionality called *lens* in computer science, take
+a look at the [excellent documentation on lenses](https://arrow-kt.io/learn/immutable-data/lens/) from the 
+[arrow-project](https://arrow-kt.io) before reading on.
 
-fritz2 also offers the method `lensOf()` for a short-and-sweet-experience, which accepts a getter- and 
+A `Lens` is basically a way to describe the relation between an outer and inner entity in a structure. It focuses on the
```

Comment:
I prefer the old structure - this is really just the teasing teaser, not the 100% in detail explanation.
In the upcoming dedicated section it is the right place to describe the details - at this place, we should omit them to focus on the big picture.